types/module/core_module_test.go

```diff
@@ -1,4 +1,4 @@
-package module
```

Comment:
This is actually importes for tests. Could it stay public?

Reply:
what tests?

Reply:
besides the ones that are in this packages?  Thats the only usage i'm seeing